seb_openedx/seb_keys_sources.py

```diff
@@ -32,5 +33,16 @@ def from_global_settings(course_key):
     return None
 
 
+def from_site_configuration(course_key):
+    """
+    Get SEB keys from djangoapps.site_configuration
+    """
+    configuration_helpers = get_configuration_helpers()
+    if configuration_helpers.has_override_value('SEB_KEYS'):
+        keys_dict = configuration_helpers.get_configuration_value('SEB_KEYS')
+        return keys_dict.get(six.text_type(course_key), None)
+    return None
+
+
 # First one has precedence over second, second over third and so forth.
-ORDERED_SEB_KEYS_SOURCES = [from_global_settings, from_other_course_settings]
+ORDERED_SEB_KEYS_SOURCES = [from_global_settings, from_other_course_settings, from_site_configuration]
```

Comment:
Given that the precedence is higher in the first value, I think our current implementation should set the preference as:
1) from_other_course_settings
2) from_site_configuration
3) from_global_settings
What is your take on this? Also, perhaps this should go in a different PR once we get there

Reply:
I don't think there is a clear order for them; maybe the best would be to merge all the available keys (but for a different PR)